armi/reactor/blueprints/tests/test_gridBlueprints.py

```diff
@@ -360,7 +361,8 @@ def tearDown(self):
 
     def test_simpleRead(self):
         gridDesign = self.grids["control"]
-        _ = gridDesign.construct()
+        grid = gridDesign.construct()
+        self.assertAlmostEqual(grid.pitch, 1.2)
```

Comment:
I was thinking it would be nice to show that this works for corners up and flats up.

I was think you could copy/paste this test, and just search/replace `"geom: hex_corners_up"` with `"geom: hex`.

Thoughts?